doc/doom_nvim.txt

```diff
@@ -227,10 +224,10 @@ Specifies the root directory of Doom Nvim.
 ==============================================================================
 AUTOCOMPLETE                                  *doom_nvim_options_autocomplete*
 
-Doom Neovim uses the built-in LSP as completion engine on Neovim Nightly.
-Use <c-n> to select the next suggestion.
+Doom Neovim uses the built-in LSP as completion engine in Neovim 0.5.0.
+Use <C-n> to select the next suggestion.
 
-Autocompletion for parenthesis can be disabled by disabling the autopairs
+Autocompletion for parenthesis can be disabled by disabling the pears.nvim
```

Comment:
We aren't using pears anymore for autopairs functionality, this line should be reverted :slightly_smiling_face: